### ISSUE TITLE
Support Azure OpenAI?

### ISSUE THREAD

**Issue by Davidasx** [microsoft/windows-ai-studio#133]
Hello, I currently have some models on Azure AI. Can the Azure AI Foundry models be integrated here too?
Thank you.

**Comment by sffamily** [comment]
Yes, it can be added as "Custom model", as long as you find and paste the key info your model deployment details page on AI Foundry: endpoint url, model name and Access Key.

Entry to add custom model: hovering over the left treeview's "MODELS" line, click "+"; under model choice list in the playground there is "+ Add Model"; on top line of Model Catalog there is also "+ Add Model". Give it a try and see how it works.

(You must make a model deployment first from Azure AI Studio.)

**Comment by Davidasx** [comment]
I know, but it's not so.convenient, isn't it. An independent entry would be better.
Thanks for your patience.
